Makefile

```diff
@@ -5,13 +5,14 @@ DESTDIR ?= /usr/local/bin
 
 LIBC ?= musl
 KBC ?=
-RESOURCE_PROVIDER ?= kbs
+RESOURCE_PROVIDER ?=
```

Comment:
The `kbs` here means actually legacy `cc-kbc`. When I coded this, I want to let the `Makefile` to have ability to distinguish the following two make scenarios so I leave out the `kbs` from default

1. build CDH with only offline-fs-kbc
2. build CDH with offline-fs-kbc and cc-kbc (feature `kbs`)

If we keep `RESOURCE_PROVIDER ?= kbs`, then when we run `RESOURCE_PROVIDER=  make build` the `RESOURCE_PROVIDER` will still be kbs as the empty env will be treated as not-set.

But overall, I am not sure if this need is reasonable.

Reply:
With #425 I think most of the platforms will want `RESOURCE_PROVIDER` to be `kbs`. For testing we are hopefully going to use cc_kbc with sample attester rather than `offline_fs_kbc`. We probably should still give an option to build with only `offline_fs_kbc`, but I wouldn't make it the default.

Reply:
Sounds reasonable. Let add a separate flag to build it with only `offline_fs_kbc`